bibtex-actions.el

```diff
@@ -417,10 +421,67 @@ TEMPLATE."
 ;; something equivalent.
 (defun bibtex-actions-get-key-org-cite ()
   "Return key at point for org-cite citation-reference."
-  (when (eq major-mode 'org-mode)
-    (let ((elt (org-element-context)))
-      (if (eq (car elt) 'citation-reference)
-          (org-element-property :key elt)))))
+  (when-let (((eq major-mode 'org-mode))
+             (elt (org-element-context)))
+    (pcase (org-element-type elt)
+      ('citation-reference
+       (org-element-property :key elt))
+      ('citation
+       (org-cite-get-references elt t)))))
+
+;;; Embark
+
+(defun bibtex-actions-citation-key-at-point ()
+  "Return citation keys at point as string for `embark'."
+  (when-let ((key (or (bibtex-actions-get-key-org-cite)
+                      (bibtex-completion-key-at-point))))
+    (cons 'citation-key (if (listp key) (string-join key ";") key))))
+
+(with-eval-after-load 'embark
+  (add-to-list 'embark-target-finders 'bibtex-actions-citation-key-at-point)
+  (add-to-list 'embark-keymap-alist '(citation-key . bibtex-actions-embark-map))
```

Comment:
Minor thing, but does the singular "key" suggest it won't work on multi-cites, like:

``` latex
\cite{doe,smith}
```
``` org
[cite:@doe;@smith]
```

Reply:
`bibtex-actions-get-key-org-cite` supports multiple keys in org mode if the point is at the citation object, e.g.

```
[cite‸:@doe;@smith]
```

Reply:
I was just wondering about the name. But it's a minor thing, as I said.